projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-form/admin-badges-form.component.scss

```diff
@@ -0,0 +1,19 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
+.admin-badges-form {
+  nb-card-body {
+    .form-fields {
+      label, nb-toggle, button {
```

Comment:
Why are we applying styles to nb-toggle?

Reply:
removed